### DISCUSSION TITLE
개발팀 프로젝트는 스포츠팀을 운영과 동일하다고 합니다.

### DISCUSSION THREAD

**Discussion by sokojh** [heading1/WYLSBingsu#13] [General]
# 서론

[Okky](https://okky.kr)라는 곳에서 10년차 개발자분이 적으신 글을 봤었는데(지금은 삭제되었나봅니다 안보이네요)

현업에서 같은 회사의 팀으로서 일할때 항상 축구팀을 결성하며 경기를 하는 것 같다고하는 글이였습니다. 

저도 축구에대해 많은걸 알지 못하지만,  

팀의 사기와 팀워크는 그 경기를 이길수 있는지 없는지를 판가르는

가장 중요한 점이라는 것은 초등학생때도 이해하고 있었습니다.

 어릴적 친구들과 함께 팀을 만들어 경기를 하려고 하면 가장 좋은사기를 가지고있는 리더와 팀워크가 잘되는 팀을 만든 친구들이 

항상 이겼다는 것은 경험으로 알고 있습니다.

# 그래서?

그래서 저는 팀 프로젝트를 하려고하면 정확한 목표의식과 좋은 팀워크 그리고 사기가 중요하다고 생각합니다.

모두가 같은 목표의식을 가지고 있다면 우리팀은 인간이 가진 가장 강력한 힘인 `상상`이라는 것을 사용 할 수 있습니다.

`상상`은 우리가 목표를 정한 것을 반드시 이뤄낼수 있다는 상상하는것입니다. 

# 조잘거리는 시간

또한 좋은 팀워크를 가지려면 서로 친해야됩니다. 

서로를 잘 알지 못하고 진행한다면 그것 또한 팀워크가 무너질 위험이 있다는 신호입니다.

그러니 저희는 언제든 `조잘거리는시간`이 낭비라 생각하지 않고 열린 마음으로 

대화를 했으면 좋겠습니다 👏

**Comment by jinux127** [comment]
좋은 생각입니다! 서로를 이해하는 만큼 프로젝트가 수월하게 진행되는 것 같아요😃

**Reply by sokojh** [reply]
맞습니다 ㅎㅎ 조잘거리는타임 자주 해야겠네요 ㅋㅋ
